packages/ses/src/error/note-log-args.js

```diff
@@ -118,49 +126,69 @@ export const makeLRUCacheMap = budget => {
     return cell;
   };
 
+  /**
+   * @param {K} key
+   */
   const has = key => touchCell(key) !== undefined;
   freeze(has);
 
+  /**
+   * @param {K} key
+   */
   // TODO Change to the following line, once our tools don't choke on `?.`.
   // See https://github.com/endojs/endo/issues/1514
-  // const get = key => touchCell(key)?.value;
+  // const get = key => touchCell(key)?.data?.get(key);
   const get = key => {
     const cell = touchCell(key);
-    return cell && cell.value;
+    return cell && cell.data && cell.data.get(key);
   };
   freeze(get);
 
+  /**
+   * @param {K} key
+   * @param {V} value
+   */
   const set = (key, value) => {
     if (budget >= 1) {
+      /** @type {LRUCacheCell | undefined} */
       let cell = touchCell(key);
```

Comment:
Actually I just realized `spliceOut` will also throw if the key was previously pushed out since the cell will be self-linked.